Sources/MapboxNavigation/NavigationMapView.swift

```diff
@@ -821,7 +827,17 @@ open class NavigationMapView: UIView {
             lineLayer = LineLayer(id: layerIdentifier)
             lineLayer?.source = sourceIdentifier
             lineLayer?.lineColor = .constant(.init(traversedRouteColor))
+        }
+        
+        let routeCasingLayerIdentifier = route.identifier(.routeCasing(isMainRoute: true))
```

Comment:
Also update the traversed route layer. Because users could modify the route casing layer property, the traversed route layer should have same property values as the main route casing layer, except the line color.